Common/src/main/java/vazkii/botania/common/block/ModBlocks.java

```diff
@@ -206,20 +208,31 @@
 	public static final Block livingrockBrickCracked = new BlockMod(BlockBehaviour.Properties.copy(livingrock));
 	public static final Block livingrockBrickMossy = new BlockMod(BlockBehaviour.Properties.copy(livingrock));
 
-	// TODO 1.16+ livingwood/dreamwood should support leaves?
-	public static final Block livingwood = new BlockMod(BlockBehaviour.Properties.of(Material.WOOD).strength(2).sound(SoundType.WOOD));
-	public static final Block livingwoodPlanks = new BlockMod(BlockBehaviour.Properties.copy(livingwood));
-	public static final Block livingwoodPlanksMossy = new BlockMod(BlockBehaviour.Properties.copy(livingwood));
-	public static final Block livingwoodFramed = new BlockMod(BlockBehaviour.Properties.copy(livingwood));
-	public static final Block livingwoodPatternFramed = new BlockMod(BlockBehaviour.Properties.copy(livingwood));
-	public static final Block livingwoodGlimmering = new BlockMod(BlockBehaviour.Properties.copy(livingwood).lightLevel(s -> 12));
+	public static final Block livingwoodLog = new RotatedPillarBlock(BlockBehaviour.Properties.of(Material.WOOD).strength(2).sound(SoundType.WOOD)); // TODO color
```

Comment:
what does this todo mean?

Reply:
Oh, that was from @Hubry, i had to ask about that as well a while ago. iirc it's for correctly setting the map color for the logs, since the vanilla logs have a different color depending on if they're vertical or horizontal